src/Psl/Str/Byte/reverse.php

```diff
@@ -9,14 +9,5 @@
  */
 function reverse(string $string): string
 {
-    $lo = 0;
-    $hi = namespace\length($string) - 1;
-
-    for (; $lo < $hi; $lo++, $hi--) {
-        $temp        = $string[$lo];
-        $string[$lo] = $string[$hi];
-        $string[$hi] = $temp;
-    }
-
-    return $string;
+    return strrev($string);
```

Comment:
please add `use function strrev` statement at the top :) 

Also, if you could add a line in the docblock explaining what this function does

Reply:
Done!